exercises/05.actions/05.solution.history-revalidation/server/app.js

```diff
@@ -13,54 +15,54 @@ import { shipDataStorage } from './async-storage.js'
 
 const PORT = process.env.PORT || 3000
 
-const app = express()
+const app = new Hono()
+
 app.use(compress())
-// this is here so the workshop app knows when the server has started
-app.head('/', (req, res) => res.status(200).end())
```

Comment:
I'm thinking we're ok without this.

What you had before meant that when the browser requested the root of the app you'd just get an empty response instead of the index.html.